frontend/src/router/index.ts

```diff
@@ -25,6 +26,11 @@ const routes: Array<RouteRecordRaw> = [
     name: "register-verification",
     component: RegisterVerificationView,
   },
+  {
+    path: "/home",
```

Comment:
This should be `/` and we should make a condition if the user logged in or not and display the component based on it.